examples/grid_example/public/app.tsx

```diff
@@ -180,24 +167,24 @@ export const GridExample = ({ coreStart }: { coreStart: CoreStart }) => {
                     })}
                     options={[
                       {
-                        id: 'VIEW',
+                        id: 'view',
```

Comment:
I switched the IDs so that they match the **dashboard** view mode, which we can then modify to send into the `GridLayout` component - this is just more inline with how it will work in the real Dashboard plugin